src/requirements.yml

```diff
@@ -11,6 +11,9 @@
   src: https://github.com/cisagov/ansible-role-clamav
 - name: cloudwatch_agent
   src: https://github.com/cisagov/ansible-role-cloudwatch-agent
+- name: harden
+  src: https://github.com/cisagov/ansible-role-hardening-2
+  version: cisagov-changes
```

Comment:
Please add a TODO comment here with a link to an issue so that this can be reverted to the default branch of `cisagov/ansible-role-hardening-2` at an appropriate future date.

Reply:
[4feb302](https://github.com/cisagov/ubuntu-server-packer/pull/15/commits/4feb302f8b03006e30c3b3c8263be9dd369990f8)

Reply:
Good, but you still have to satisfy the markdown linter.  You should be running pre-commit in your local environment so that things like this are caught before they get pushed to GitHub.